docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-article-body/index.scss

```diff
@@ -0,0 +1,37 @@
+@use 'uswds-core' as *;
+@use 'usa-list' as list;
+@use 'usa-prose' as prose;
+@use 'nci-heading' as heading;
+
+.cgdp-article-body {
```

Comment:
Woah there. We should not be doing this right now. We will add usa-prose when we update the WYSIWYG. Was there a specific reason for pulling it in?

(the `.cgdpl` legacy css is going to be fighting with uswds styles in a mostly bad way.)